src/pages/academy/sourcereel/Sourcereel.tsx

```diff
@@ -299,29 +324,62 @@ const Sourcereel: React.FC<SourcereelProps> = props => {
     id: SideContentType.envVisualizer
   };
 
+  const workspaceHandlers = useMemo(() => {
+    return {
+      handleBrowseHistoryDown: () => dispatch(browseReplHistoryDown(workspaceLocation)),
+      handleBrowseHistoryUp: () => dispatch(browseReplHistoryUp(workspaceLocation)),
+      handleReplValueChange: (newValue: string) =>
+        dispatch(updateReplValue(newValue, workspaceLocation)),
+      handleDeleteSourcecastEntry: (id: number) =>
+        dispatch(deleteSourcecastEntry(id, 'sourcecast')),
+      // SourcereelControlbar handlers
+      handleResetInputs: (inputs: Input[]) => dispatch(resetInputs(inputs, workspaceLocation)),
+      handleSaveSourcecastData: (
+        title: string,
+        description: string,
+        uid: string,
+        audio: Blob,
+        playbackData: PlaybackData
+      ) => dispatch(saveSourcecastData(title, description, uid, audio, playbackData, 'sourcecast')),
```

Comment:
we can consider abstracting these `'sourcecast'` strings into a single constant at the top of the file

Reply:
Resolved in dcf2e8ee6a8745999639d0cc936e715face5b234.